packages/cozy-procedures/src/components/overview/PersonalDataFullyCompleted.jsx

```diff
@@ -23,10 +23,11 @@ const PersonalDataFullyCompleted = ({ navigateTo, personalData, t }) => {
   } = personalData
   return (
     <InlineCard
-      className="u-c-pointer u-p-1 u-flex u-flex-column"
+      className="u-c-pointer u-ph-1 u-pt-half u-pb-1 u-flex u-flex-column"
```

Comment:
Ideally though, we would do it like the DocumentsFullyCmpleted: horizontal padding of 1 and vertical padding of half on the container, and vertical padding of half on all the children :)

Reply:
thanks !